README.md

```diff
@@ -334,7 +316,110 @@ You can access a vector of pointers to the parsed options in the original order
 If `--` is present in the command line that does not end an unlimited option, then
 everything after that is positional only.
 
-#### Getting results 🚧
+#### Validators
+Validators are structures to check or modify inputs, they can be used to verify that an input meets certain criteria or transform it into another value.  They are added through an options `check` or `transform` functions.  They differences between those two function are that checks do not modify the input whereas transforms can and are executed before any Validators added through `check`.
+
+CLI11 has several Validators built in that perform some common checks
+
+-   `CLI::IsMember(...)`: 🚧 Require an option be a member of a given set. See below for options.
+-   `CLI::Transformer(...)`: 🚧 Modify the input using a map. See below for options.
+-   `CLI::CheckedTransformer(...)`: 🚧 Modify the input using a map, and verify that the input is either in the set or already one of the outputs of the set. See below for options.
```

Comment:
What about the `->unchecked` idea?

Reply:
There is an active(false) function on validators now.  That is the same concept,  It could be renamed as checked(), unchecked(), is_checked(), if you think that would read better and be more intuitive.